packages/taquito-rpc/src/types.ts

```diff
@@ -1148,6 +1148,20 @@ export interface PackDataParams {
   gas?: BigNumber;
 }
 
+export interface ticketTokenParams {
+  ticketer: string;
+  content_type: MichelsonV1Expression;
+  content: MichelsonV1Expression;
+}
+
+export interface allTicketBalancesResponse {
+  ticketer: string;
+  content_type: MichelsonV1Expression;
+  content: MichelsonV1Expression;
+  amount: string;
+}
+[];
```

Comment:
I think instead we should make a single `TicketBalance` type, and have `AllTicketBalancesResponse` be an array of that type